packages/blade/src/components/Collapsible/CollapsiblePanel.web.tsx

```diff
@@ -121,8 +121,12 @@ const CollapsiblePanel = ({ children }: CollapsiblePanelProps): ReactElement =>
       onTransitionEnd={onTransitionEnd}
     >
       <Box
-        // need margin here to prevent jank
-        marginY="spacing.5"
+        /**
+         * Need a margin inside the outside wrapper so this is
+         * included in height calculations and prevents jank
+         */
+        marginTop={direction === 'bottom' ? 'spacing.5' : 'spacing.0'}
```

Comment:
have we confirmed this with design? because there might be cases where consumers might need lesser margin top. this is 16px not sure if we have this standardised from design side?

Reply:
Yes, this is covered in the [design tokens spec](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-629510&t=iylL78shTIJyxxW7-4)

<img width="545" alt="image" src="https://github.com/razorpay/blade/assets/6682655/be1789bb-c2c5-4938-8fcc-2b92e3a6c8d7">